### PR TITLE
50 refactor logos component to make it reusable

### PR DESCRIPTION
**Pull request by SevinjFeyzi**

### CODE REVIEW

File: frontend/src/components/Logo/Logo.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<<<<<<< Updated upstream
```
Comment (uparkalau) [inline frontend/src/components/Logo/Logo.jsx:1]: review your code . what is ">>>>>>> Stashed changes" you did not test your code after merge.

**Review by uparkalau — CHANGES_REQUESTED** [review]
review your code again. this is a lot of merge artifacts

File: frontend/src/.App.jsx.swo
Comment (erenfn) [inline frontend/src/.App.jsx.swo:1]: do not push these files

File: frontend/src/.App.jsx.swp
Comment (erenfn) [inline frontend/src/.App.jsx.swp:1]: Same. Do not push these files

File: frontend/src/components/Logo/TestLogo.jsx
Comment (erenfn) [inline frontend/src/components/Logo/TestLogo.jsx:1]: Is this file going to be used? If it is just for testing remove it.

File: frontend/src/App.jsx [hunk 2]
```diff
@@ -6,13 +6,17 @@ import PasswordResetPage from "./scenes/login/PassswordResetPage";
 import ForgotPasswordPage from "./scenes/login/ForgotPasswordPage";
 import CheckYourEmailPage from "./scenes/login/CheckYourEmailPage";
 import SetNewPasswordPage from "./scenes/login/SetNewPassword";
+import TestLogo from "./components/Logo/TestLogo";
```
Comment (erenfn) [inline frontend/src/App.jsx:9]: If TestLogo is just for testing, use the actual Logo instead. And I am not sure what Logo is doing in App.jsx

**Review by erenfn — CHANGES_REQUESTED** [review]

File: frontend/src/scenes/home/Home.jsx [hunk 3]
```diff
@@ -1,10 +1,20 @@
...
 import "./Home.css"
-const Home = () => {
 
+const Home = () => {
+    const logo = {
+        src: LogoBW,
+        alt: 'Introflow logo',
+        className: 'logo-item',
+      };
+      console.log('Home logo object:', logo);
```
Comment (uparkalau) [inline frontend/src/scenes/home/Home.jsx:13]: Delete all console.log

**Review by uparkalau — CHANGES_REQUESTED** [review]

File: frontend/src/scenes/home/Home.jsx (hunk 3 above)
Comment (SevinjFeyzi) [reply frontend/src/scenes/home/Home.jsx:13]: Done!

File: frontend/src/components/Logo/Logo.jsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './LogoStyles.css';
+
+const Logo = ({ logo }) => {
+  console.log('Logo prop:', logo);
```
Comment (uparkalau) [inline frontend/src/components/Logo/Logo.jsx:6]: Check all code please

**Review by uparkalau — CHANGES_REQUESTED** [review]

**Review by uparkalau — APPROVED** [review]
You can merge your code into develop

File: frontend/src/components/Logo/Logo.jsx
Comment (swoopertr) [inline frontend/src/components/Logo/Logo.jsx:1]: This logo component props may have "width" and "height" in the propTypes.

File: frontend/src/components/Logo/LogoStyles.css
Comment (swoopertr) [inline frontend/src/components/Logo/LogoStyles.css:1]: logoImage class never used in project. If it won't used. You can delete.

File: frontend/src/scenes/home/Home.jsx
Comment (swoopertr) [inline frontend/src/scenes/home/Home.jsx:1]: I think this is the usage how to use logo component and sending object as a props. 
It looks like giant logo in the screen :). We definitely need width and height in props.

File: frontend/src/App.jsx
Comment (swoopertr) [inline frontend/src/App.jsx:1]: in 20th row <Logo/> component doing nothing. You can remove it or use properly.

File: frontend/src/App.jsx [hunk 5]
```diff
@@ -6,13 +6,18 @@ import PasswordResetPage from "./scenes/login/PassswordResetPage";
...
+
+
 
 
 function App() {
   // const isAuthenticated = useSelector((state) => state.auth.isLoggedIn); // TODO: redux implementation
   const isAuthenticated = true;// emulate login
   return (
     <>
+<Logo/>
```
Comment (uparkalau) [inline frontend/src/App.jsx:20]: delete it from here

File: frontend/src/components/Logo/LogoStyles.css [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+  
+  .logo-item {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+  
+  .logoImage {
+
+    height: 75px;
```
Comment (erenfn) [inline frontend/src/components/Logo/LogoStyles.css]: For later, you may want to do the Logo more adaptable to different containers or window sizes and not hardcode the dimensions (px)

But for now it is fine

**Pull request merged**